assets/cluster-version-operator/cluster-version-operator-deployment.yaml

```diff
@@ -42,6 +42,24 @@ spec:
 {{ if .MasterPriorityClass }}
       priorityClassName: {{ .MasterPriorityClass }}
 {{ end }}
+      initContainers:
```

Comment:
We need to set the security context for this container.

Reply:
We also need resource requests and limits support for all containers.

Reply:
Added